### PR TITLE
[activesupport] Update links to use https instead of http [ci skip]

### PR DESCRIPTION
**Pull request by yhirano55**
### Summary

- Update links to use `https` instead of `http` in activesupport.
- All changes are only in comment blocks. So I've skipped CI.

### CODE REVIEW

File: activesupport/lib/active_support/multibyte/unicode.rb [hunk 1]
```diff
@@ -265,7 +265,7 @@ def tidy_bytes(string, force = false)
       #   Default is ActiveSupport::Multibyte::Unicode.default_normalization_form.
       def normalize(string, form = nil)
         form ||= @default_normalization_form
-        # See http://www.unicode.org/reports/tr15, Table 1
+        # See https://www.unicode.org/reports/tr15, Table 1
```
Comment (kamipo) [inline activesupport/lib/active_support/multibyte/unicode.rb]: `www.` is unneeded.

```
% curl -v -I https://www.unicode.org/reports/tr15
*   Trying 216.97.88.9...
* Connected to www.unicode.org (216.97.88.9) port 443 (#0)
* TLS 1.2 connection using TLS_DHE_RSA_WITH_AES_256_GCM_SHA384
* Server certificate: unicode.org
* Server certificate: COMODO RSA Organization Validation Secure Server CA
* Server certificate: COMODO RSA Certification Authority
* Server certificate: AddTrust External CA Root
> HEAD /reports/tr15 HTTP/1.1
> Host: www.unicode.org
> User-Agent: curl/7.43.0
> Accept: */*
> 
< HTTP/1.1 301 Moved Permanently
HTTP/1.1 301 Moved Permanently
< Date: Tue, 22 Aug 2017 02:07:00 GMT
Date: Tue, 22 Aug 2017 02:07:00 GMT
< Server: Apache
Server: Apache
< Location: https://unicode.org/reports/tr15/
Location: https://unicode.org/reports/tr15/
< Vary: Accept-Encoding
Vary: Accept-Encoding
< Content-Type: text/html; charset=iso-8859-1
Content-Type: text/html; charset=iso-8859-1

< 
* Connection #0 to host www.unicode.org left intact
```

Comment (kamipo) [inline activesupport/lib/active_support/multibyte/unicode.rb]: Did you confirmed the page? https://www.unicode.org/reports/tr15/
Looks like that the original link is the right page.

Comment (yhirano55) [inline activesupport/lib/active_support/multibyte/unicode.rb]: @kamipo Thanks for you comments. I'll check all links again, and fix it soon.

**Comment by yhirano55** [comment]
@kamipo I've checked all links and fixed them.

- Add trailing slash to `https://www.limited-entropy.com/padding-oracle-attacks/`
- Remove trailing slash from `https://memcached.org`
- `https://www.ietf.org/rfc/rfc4122.txt` doesn't have any problems.
- It seems that contents of `www.unicode.org` is different from between http and https. So I've restored them.

**Pull request merged**